ocs_ci/ocs/ui/views.py

```diff
@@ -471,6 +478,8 @@
     ),
     "thin_sc": ('a[id="thin-link"]', By.CSS_SELECTOR),
     "gp2_sc": ('a[id="gp2-link"]', By.CSS_SELECTOR),
+    "gp2-csi_sc": ('a[id="gp2-csi-link"]', By.CSS_SELECTOR),
+    "gp3-csi_sc": ('a[id="gp3-csi-link"]', By.CSS_SELECTOR),
```

Comment:
Same locator at line 147 & 148 can be used

Reply:
These are different buttons one for "deployment" and the other for "add capacity".